src/controllers/playback/video/index.js

```diff
@@ -1466,15 +1417,9 @@ import { appRouter } from '../../../components/appRouter';
             passive: true
         });
 
-        if (browser.touch) {
-            dom.addEventListener(view, 'dblclick', onDoubleClick, {});
-        } else {
-            const options = { passive: true };
-            dom.addEventListener(view, 'dblclick', (e) => {
-                if (e.target !== view) return;
```

Comment:
Removing this line breaks #2006
As I said on Matrix, need to add `pointer-events: none;` to `syncPlayContainer`, because the latter overlaps the view and grabs `dblclick`.